packages/client/src/workflow-client.ts

```diff
@@ -302,11 +305,14 @@ function defaultWorkflowClientOptions(): WithDefaults<WorkflowClientOptions> {
   };
 }
 
-function assertRequiredWorkflowOptions(opts: WorkflowOptions): void {
-  if (!opts.taskQueue) {
+function assertRequiredWorkflowOptions(opts?: {
```

Comment:
Why did you have to make `opts` optional? Should we check specifically for `!opts` and give a distinct error message if it is undefined?

Reply:
Good question. That `?` looks like a mistake to me. Removed.